docs/beaconchain/develop/api-reference/sdk.md

```diff
@@ -1,15 +0,0 @@
-# SDKs for Binance Chain
-
-Here the client SDK is used to connect to Binance Chain and DEX. There are libraries in a few programming languages, including:
-
-- [Go](https://github.com/bnb-chain/go-sdk)([Documentation](https://github.com/bnb-chain/go-sdk/wiki))
```

Comment:
We still have go-sdk and java-sdk, dont we?

Reply:
this page is duplicated. 
please check another file `docs/beaconchain/develop/bcsdk.md`